test/Sentry.Tests/Protocol/Exceptions/SentryStackFrameTests.cs

```diff
@@ -22,7 +22,6 @@ public void SerializeObject_AllPropertiesSetToNonDefault_SerializesValidObject()
             Vars = { { "var", "val" } },
             FramesOmitted = { 1, 2 },
             Package = "Package",
-            Platform = "Platform",
```

Comment:
Lets keep this test as it was, to ensure we can override the platform for a single stack frame.